apps/src/sites/studio/pages/scripts/stage_extras.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import StageExtras from '@cdo/apps/code-studio/components/stageExtras/StageExtras';
+
+const script = document.querySelector('script[data-extras]');
+const config = JSON.parse(script.dataset.extras);
+
+ReactDOM.render(
+  <StageExtras
+    stageNumber={config.stageNumber}
+    nextLevelPath={config.nextLevelPath}
+  />,
+  document.querySelector('.stage-extras')
```

Comment:
any reason not to use an id instead of a className, as I assume we expect only one.